mailer.js

```diff
@@ -26,6 +27,28 @@ module.exports = function (log) {
     return 'href="' + url + '" style="color: #0095dd; text-decoration: none; font-family: sans-serif;"'
   }
 
+  function constructLocationString (location) {
+    // construct the location string from the location object
+    var locationString = ''
+    if (location !== null) {
+      if (location.city) {
+        locationString += location.city + ', '
+      }
+      locationString += location.country + ' ' + '(estimated)'
```

Comment:
@vladikoff we discussed this in the mtg with @vbudhram and @ryanfeeley. `Estimated` is what we want to try out as the language for `not 100% accurate`. Will add the entire string to gettext() 👍